__tests__/server.test.ts

```diff
@@ -0,0 +1,35 @@
+import { createApolloServer } from '../src/index'
+import { beforeEach, afterAll, expect } from '@jest/globals'
+import request from 'supertest'
+
+const queryData = {
+    query: `query Query($healthcareProfessionalId: ID!) {
+        healthcareProfessional(id: $healthcareProfessionalId) {
+          id
+        }
+      }`,
+    variables: {
+        healthcareProfessionalId: '1'
+    }
+}
+
+describe('query healthcareProfessionalById', () => {
+    let server: { stop: () => void }, url: string
+
+    beforeEach(async () => {
+        ({ server, url } = await createApolloServer(0))
+    })
+    afterAll(done => {
+        server.stop()
+        done()
+    })
+    
+    it('returns an error if a healthcare professional is not found', async () => {
```

Comment:
This is a good failure test. 
We should also always include a "happy path" test which is the simplest success test for this endpoint.